tbx/people/models.py

```diff
@@ -56,36 +65,54 @@ class PersonPage(ColourThemeMixin, SocialFields, Page):
     promote_panels = [
         MultiFieldPanel(Page.promote_panels, "Common page configuration"),
         MultiFieldPanel(SocialFields.promote_panels, "Social fields"),
+        FieldPanel("related_teams", widget=forms.CheckboxSelectMultiple),
     ]
 
     @cached_property
     def author_posts(self):
-        # return the blogs writen by this member
-        author_snippet = Author.objects.get(person_page__pk=self.pk)
+        # Get the BlogPages associated with this page's authors
+        blog_pages = (
+            BlogPage.objects.live().filter(authors__page=self).order_by("-date")
+        )
 
-        # format for template
+        # Format for template
         return [
             {
                 "title": blog_post.title,
                 "url": blog_post.url,
                 "author": blog_post.first_author,
                 "date": blog_post.date,
+                "tags": blog_post.related_teams,
             }
-            for blog_post in BlogPage.objects.live()
-            .filter(authors__author=author_snippet)
-            .order_by("-date")
+            for blog_post in blog_pages
         ]
 
     @cached_property
     def related_works(self):
-        # Get the latest 2 work pages by this author
-        works = (
-            HistoricalWorkPage.objects.filter(authors__author__person_page=self.pk)
+        """Returns work pages authored by the person, giving preference to Work rather than Historical work pages"""
+        # Get the latest 3 work pages by this author
+        recent_works = (
+            WorkPage.objects.filter(authors__author__person_page=self.pk)
             .live()
             .public()
             .distinct()
-            .order_by("-date")[:2]
+            .order_by("-date")[:3]
         )
+
+        remaining_slots = 3 - len(recent_works)
+
+        if remaining_slots > 0:
+            # Get the latest 3 historical work pages by this author iff necessary
```

Comment:
```suggestion
            # Get the latest 3 historical work pages by this author if necessary
```

Reply:
I think in this case it looks like a typo as it is part of a comment - so writing it more verbosely is probably the best thing.